docs/source/get-involved/gsoc/projects_2025.md

```diff
@@ -0,0 +1,251 @@
+# GSoC NIU Projects 2025
+
+Below you can find our list of projects for GSoC 2025 (aka our "Ideas List"). 
+
+<!-- There are three main milestones in a GSoC project: the community bonding period, the start of coding, and the coding end.
```

Comment:
Question for reviewer(s):

Is this commented out section worth keeping?

At first I thought it was useful information, but then I think it may be repetitive with the GSoC docs. It feels like a maintenance burden to make sure this is updated and in sync with them if it changes for other years.